homeassistant/components/advantage_air/climate.py

```diff
@@ -263,7 +204,7 @@ def hvac_modes(self):
     @property
     def device_state_attributes(self):
         """Return additional attributes about Zone."""
-        return self.coordinator.data["aircons"][self.ac_key]["zones"][self.zone_key]
+        return self._zone
```

Comment:
We want the items in the state attributes to be set explicitly so we can see what the keys are.

Reply:
I have removed the property as I don't believe its required.